proposals/stage-1/PROJECT_ONBOARDING/PROJECT_ONBOARDING.md

```diff
@@ -0,0 +1,22 @@
+# Onboarding Checklist
+
+This is an informational checklist to help projects onboard into the OpenJS Foundation. If you have any questions or need help, the OpenJS Foundation CPC is available to assist.
+
+- [ ] Adopt the OpenJS Foundation Code of Conduct
```

Comment:
Is this a requirement prior to joining, or is a commitment to do so sufficient? I have a project that i very much want to join and use this CoC, but prior to living in a neutral home i may be unable to easily make these sorts of changes.

Reply:
This is a checklist for projects joining.  Not things that need to be done in advance.  I believe a commitment to do so, and then doing it with X time after joining is what we had in mind.

Reply:
Perfect, that clarification helps :-)

Reply:
added some clarifying prose!